src/shared/api-errors/api-mutation-error.js

```diff
@@ -0,0 +1,54 @@
+import { FetchError } from '@dhis2/app-service-data/build/es/engine/index.js'
```

Comment:
I know this import is not ideal, but this is not exported from `app-runtime` or the `app-service-data` indexes.